src/one/nio/serial/Serializer.java

```diff
@@ -133,6 +137,15 @@ public static int sizeOf(Object obj) throws IOException {
     }
 
     public static Object deserialize(byte[] data) throws IOException, ClassNotFoundException {
-        return new DeserializeStream(data).readObject();
+        try {
+            return new DeserializeStream(data).readObject();
+        } catch (SerializerNotFoundException e) {
```

Comment:
but in fact I don't see big difference with https://github.com/odnoklassniki/one-nio/blob/master/src/one/nio/rpc/RpcClient.java#L49 
Is it?